samples/spring-eventsourced-customer-registry/src/main/java/customer/api/Address.java

```diff
@@ -0,0 +1,18 @@
+package customer.api;
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Address {
+  public String street;
+  public String city;
+
+  @JsonCreator
+  public Address(@JsonProperty("street") String street, @JsonProperty("city") String city) {
```

Comment:
```suggestion
  public Address(String street, String city) {
```

Brave enough to remove these then? :D